vyper/builtin_functions/functions.py

```diff
@@ -106,15 +111,26 @@ class _BuiltinFunction:
     def __repr__(self):
         return f"builtin function {self._id}"
 
+    def infer_kwarg_types(self, node):
+        if node.keywords:
```

Comment:
not sure why this branch is here. clearest options IMO are to always return types for all potential kwargs (`self._kwargs`) or just for the mentioned keywords (`node.keywords`); neither requires a branch.